### ISSUE TITLE
Presenting these results at conferences

### ISSUE THREAD

**Issue by adele-morrison** [willaguiar/ASC_and_heat_transport#33]
Hi folks, 

How do we feel about presenting this work at the Canberra July workshop? I don't have much other work to present at the moment, so would be happy to present if others are happy with that?

Here's a suggested title and abstract. I'm not quite sure about the title (because see the last sentence - this is not actually what we're investigating here. But it is a bit catchier than "The relationship between the Antarctic Slope Current and ocean heat transport towards Antarctica").

Any suggested changes?
_____

**Does the Antarctic Slope Current control ocean heat transport towards Antarctica?**

Increased ocean heat transport towards Antarctica directly drives melting of ice shelves, leading to sea level rise and reduced dense water formation. A common dynamical assumption is that poleward heat transport across the Antarctic continental slope is controlled by the strength of the Antarctic Slope Current (ASC), which is thought to act as a barrier to heat transport. However, the relationship between poleward heat transport and the ASC has not been examined in detail. Here, using a global, eddy-rich ocean – sea ice simulation, we find that the strength of the relationship varies significantly around Antarctica, with local correlations between ASC speed and cross-slope heat transport ranging from r2 < 0.1 to r2 > 0.8. In the temporal domain, the relationship between the ASC and heat transport is strongest on seasonal and interannual timescales, with surprisingly low correlations at high frequency, eddy timescales. Our results suggest that the relationship between ASC strength and poleward heat transport may not be as simple as is often assumed. Even for regions and timescales with a strong relationship, it remains an open question whether a strong ASC is dynamically limiting the heat transport, or whether both quantities are responding concurrently to external forcing.

**Comment by taimoorsohail** [comment]
This sounds like a great idea Adele! I actually like the title, it opens the door for more discussion. This was our original motivation for this study, and while we found a correlation, we still need to do more to discover whether the ASC drives CSHT (or vice versa), or if there is an external driver controlling both. I'm happy for you to submit it.

**Comment by PaulSpence** [comment]
Thanks Adele!

**Comment by ongqingyee** [comment]
Thanks Adele, my only suggestion would be some mention of variations across different regimes, maybe "Our results suggest that the relationship between ASC strength and poleward heat transport may not be as simple as is often assumed, _with the strength of the correlation also varying across different dynamical regimes." 

Happy with everything else!

**Comment by adele-morrison** [comment]
Yes, good suggestion Ellie. Problem is it's currently at 198 words...

**Comment by ongqingyee** [comment]
Maybe instead "Here, using a global, eddy-rich ocean – sea ice simulation, we find that the strength of the relationship varies significantly ~~around Antarctica~~ _**across different ASC regimes,**_ with local correlations between ASC speed and cross-slope heat transport ranging from r2 < 0.1 to r2 > 0.8."

But on second reading the original line does have the spatial information so I'm ok either way. My bad!

**Comment by adele-morrison** [comment]
Perfect, 200 words exactly!

**Comment by taimoorsohail** [comment]
Hi all, I am going to the Challenger conference in the UK in September, and I thought I could also present this work to folks there (I imagine there will be a good presence from BAS and NOC, for example). Are we all happy with this? I will adapt the Abstract above and submit it (deadline is today!)

**Comment by adele-morrison** [comment]
Attaching my slides from the OMO meeting here. [Morrison_ASC_heat_transport.pdf](https://github.com/user-attachments/files/16105276/Morrison_ASC_heat_transport.pdf)

**Comment by ongqingyee** [comment]
> Attaching my slides from the OMO meeting here. [Morrison_ASC_heat_transport.pdf](https://github.com/user-attachments/files/16105276/Morrison_ASC_heat_transport.pdf)

For the Getz and Totten transects, it looks like the velocity structure in the Totten remains unchanged between strong and weak ASC, while the ASC penetrates deeper in the Getz while it speeds up. Velocities on the Totten are also much greater than on Getz.

Could there be a threshold where for a weaker ASC, the CSHT and ASC speed are correlated, while for stronger ASC there is little difference? i.e. for ASC stronger than a certain threshold, the along-isopycnal pathway for CDW intrusions is still blocked, at least on an interannual timescale? I guess this is kind of inline with what you were thinking @adele-morrison, where the changing sign of the ASC could make a difference.

**Comment by adele-morrison** [comment]
Sounds like a good hypothesis for testing @ongqingyee!

**Comment by willaguiar** [comment]
Hello Everyone.

I thought of submitting this abstract for the Australian Antarctic Research Conference if everyone is ok with it?

On another note, I had the opportunity to present this work here on my visit in NOC. We had lots of questions and engagement in the presentation.

Here are some of the suggestions/points raised during the presentation:

-	People found interesting that we were trying to test this core wide assumption of the ASC/ASF controls the CSHT. They raised the point that people would often have the same belief about the ACC + poleward heat transport a few years ago, but now it is shown that the ACC is more like a leaky pipe. 
-	It was suggested that instead of correlating ASC with CSHT, it might be worth to correlate the time derivative of ASC speed with the CSHT. The idea here would be that maybe we have the stronger heat transport happening while the isopycnals starts to shift upwards, then allowing more heat to enter the shelf. A flood-gate hypothesis (slowing ASC opens the gate for the CSHT).
-	It might be useful to calculate the EKE along/around the isobath and see how it changes as the ASC speed up. It would be a way to double check the eddy activity relationship with the ASC speed and test a little more thoroughly the dynamic barrier idea.
-	In turn, people were often confused about the dynamical barrier idea, and often thought that ASF might be correlated with CSHT, but not the ASC necessarily. That matches our results in some places on the shelf, but makes me think that we must be careful on how we phrase things on the manuscript for a potential reviewer.
-	Another thing that was raised was that it might be worth to check the salinity on the shelf. Since the Southern Ocean is bitter, salinity is determining the isopycnals on the model, so it could be that changes in salinity are the ones allowing CDW to enter. Personally, I was not sure if it would be possible to have clear signal there, as CDW would have a salinity signature too.
-	Another suggestion was for us to look at the heat budget spatially (Plot zonal convergence signal, separated from cross transport signal). That is because they had a sense that some of the correlations would be messed up by the zonal convergence signal. 
-	It was raised that some studies show that the baroclinic component of the ASC takes a longer time to respond to forcings, which means that there could be a delay between the ASC speed and CSHT. So it might be worth to test lagged correlations.

**Comment by willaguiar** [comment]
Hi Everyone. 

I am presenting the results of our project on AARC. Below are 2 versions of the poster RN (IF you prefer one specifically, let me know)
[Poster_AARC_V0.pdf](https://github.com/user-attachments/files/17694988/Poster_AARC_V0.pdf)


Note: I am using our photos in the poster, just like in Adele's presentation. In case you are not comfortable with your photo there, let me know.

**Comment by fabiobdias** [comment]
Thanks @willaguiar for putting this together! I really like the poster design, and the new maps with the correlations/slope simplified, nicely done! 

I don't have a strong opinion on which version, maybe I'd go with the 1st version (with Thompson's figure in it), but just for aesthetics reason as both show the same information. And one last comment, I think maybe a different (darker) colorfont in the texts inside all the light green/blue-ish boxes would make it easier to read.

**Comment by ongqingyee** [comment]
Looks good Will! Is there any way to make the two key circum-antarctic
correlation/slope plots bigger? There is still some white space below the
teal text box, and additional space between the bullet points, so I think
there is room for the nice plots to be clearer!

**Comment by willaguiar** [comment]
Hello Everyone!

I am thinking one abstract on this  to IAPSO/BACO2025. (deadline today sorry). Let me know in case you had plans for this already
